src/devices/Mcp3428/ConversionResult.cs

```diff
@@ -0,0 +1,48 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
+
+namespace Iot.Device.Mcp3428
+{
+    public readonly struct ConversionResult
+    {
+        public ConversionResult(byte channel, short rawData, ResolutionEnum resolution)
+        {
+            Channel = channel;
+            RawData = rawData;
+            Resolution = resolution;
+        }
+
+        /// <summary>
+        /// Gets the channel.
+        /// </summary>
+        /// <value>The channel.</value>
+        /// <autogeneratedoc />
+        public byte Channel { get; }
+
+        /// <summary>
+        /// Gets the raw data.
+        /// </summary>
+        /// <value>The raw data.</value>
+        /// <autogeneratedoc />
+        public Int16 RawData { get; }
+
+        /// <summary>
+        /// Gets the resolution.
+        /// </summary>
+        /// <value>The resolution.</value>
+        /// <autogeneratedoc />
+        private ResolutionEnum Resolution { get; }
+
+        public double LSBValue => Helpers.LSBValue(Resolution);
```

Comment:
this should have some comment or use full name since it's not clear what that is. Also probably the proper casing is `LsbValue` assuming `LSB` is what the industry uses as a name

Reply:
Fixed with comment. LSB (= least significant bit) is indeed the name used in the industry.

Reply:
@kmate95 - didn't relate because the type is double :-) perhaps should be called just "Value"